lib/TPP/ConvertLinalgToTpp.cpp

```diff
@@ -110,6 +134,30 @@ struct ConvertMatmulToTpp : public OpRewritePattern<linalg::MatmulOp> {
   }
 };
 
+// Convert a linalg.matmul to a tpp.matmul.
+struct ConvertFillToTpp : public OpRewritePattern<linalg::FillOp> {
```

Comment:
Do we test this pattern?

Reply:
Yes, I added `@linalg_fill_*` test cases.